sirepo/job_supervisor.py

```diff
@@ -295,6 +303,58 @@ def __db_write(self):
         sirepo.util.json_dump(self.db, path=self.__db_file(self.db.computeJid))
         return self
 
+    @classmethod
+    def _get_running_jobs(cls, want_adm=False):
+        def _get_jobs():
+            o = []
+            for i in filter(
+                    lambda x: x.db.status == job.RUNNING,
+                    cls.instances.values(),
+            ):
+                d = i.db
+                s = d.computeJobStart
+                l = d.lastUpdateTime
+                r = [
+                    d.simulationType,
```

Comment:
I don't think simulationType and simulationId are useful for end users. The SimulationName should be enough. Users think in terms of what they see, and that's the name of the simulation. End users probably don't use more than one simulationType simultaneously so this will be a wasted column.